Composer/packages/server/src/models/bot/sampler/ReservoirSampler.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { ReservoirSampler } from '@microsoft/bf-dispatcher/lib/mathematics/sampler/ReservoirSampler';
+
+import { IUtterance } from './BootstrapSampler';
+
+const MIN_SAMPLE_SIZE = 15000;
```

Comment:
same here

Reply:
if you don't make this configurable, i bet this limit would also never hit, this module will even never run?